data/json/items/armor/cloaks.json

```
@@ -148,6 +148,56 @@
       }
     ]
   },
+  {
+    "id": "state_flag",
+    "repairs_like": "sheet",
+    "type": "ARMOR",
+    "name": { "str": "state flag" },
+    "description": "A large white flag with a shield featuring an American Indian and a star which represents the state of Massachusetts.",
```

```suggestion
    "description": "A large white flag with a shield featuring an American Native and a star which represents the state of Massachusetts.",
```
editing to make it consistent with your text below.